src/libs/onlyWeb/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Only run the callback on Web and return the result as Boolean.
+ *
+ * @param {Function} callback
+ * @return {Boolean} Always false on native platforms
```

Comment:
Instead of mentioning native platforms **here**, I'd mention the native-related issue in `index.native.js`

Reply:
OK.